pallets/liquidity-pools/src/message.rs

```diff
@@ -257,13 +254,10 @@ where
 		investor: Address,
 		/// The currency in which the investment was realised
 		currency: u128,
-		/// The amount that was actually collected
+		/// The amount that was actually collected, in `currency` units
 		currency_payout: Balance,
 		/// The amount of tranche tokens received for the investment made
 		tranche_tokens_payout: Balance,
-		/// The remaining amount of `currency` the investor still has locked to
-		/// invest at a later epoch execution
-		remaining_invest_order: Balance,
```

Comment:
NOTE: This was discussed, see [DAO slack comment](https://centrifugedao.slack.com/archives/C04HCK98KLL/p1692046938307889?thread_ts=1691760753.515679&cid=C04HCK98KLL).